src/libraries/System.Linq/src/System/Linq/Average.cs

```diff
@@ -51,7 +51,7 @@ public static double Average(this IEnumerable<int> source)
                 return (double)sum / span.Length;
             }
 
-            using (IEnumerator<int> e = source.GetEnumerator())
+            using IEnumerator<int> e = source.GetEnumerator();
```

Comment:
why do we keep braces below this statement

Reply:
Thanks, removed.